getting-started/quickstart-guides/angular.md

```diff
@@ -678,9 +669,32 @@ For this, we will use the `ik-upload` component as well as a couple of event han
     fileName="test.jpg" 
     (onError)="handleUploadError($event)"
     (onSuccess)="handleUploadSuccess($event)" 
-    >
-  </ik-upload>
+    [authenticator]="authenticator"
+></ik-upload>
+```
+
+In `app.component.js` file:
+
```

Comment:
Add some useful information e.g. `you can call a secure endpoint on your backend and pass any custom request headers for authentication purpose`.

Reply:
Added text and example with custom request headers